js/utilities.js

```diff
@@ -2220,6 +2255,14 @@
         }, text);
     };
 
+    module._escapeHTMLCharacters = function (text) {
```

Comment:
We have this function in chaise:
```
function makeSafeIdAttr(string) {
            return String(string)
                .replace(/&/g, '&amp;')
                .replace(/\s/g, '&nbsp;') // any whitespace
                .replace(/</g, '&lt;')
                .replace(/>/g, '&gt;')
                .replace(/"/g, '&quot;')
                .replace(/'/g, '&#39;');
        }
```

Maybe this one you wrote should consider whitespace characters too.